packages/trpc/server/routers/publicViewer/noShow.handler.ts

```diff
@@ -7,9 +7,9 @@ type NoShowOptions = {
 };
 
 export const noShowHandler = async ({ input }: NoShowOptions) => {
-  const { bookingUid, attendees } = input;
+  const { bookingUid, attendees, noShowHost } = input;
 
-  return handleMarkNoShow({ bookingUid, attendees });
+  return handleMarkNoShow({ bookingUid, attendees, noShowHost });
```

Comment:
not related to PR but i found this was missing while working on https://github.com/calcom/cal.com/pull/15738